portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -63,6 +63,27 @@ definition {
 			entryType = "${customElementType}");
 	}
 
+	@description = "Verify that remote app of type Custom Element can be deleted"
+	@priority = "4"
+	test CustomElementCanBeDeleted {
+		property portal.acceptance = "true";
+
+		task ("Create Vanilla Counter as a Custom Element") {
+			JSONRemoteApp.addCustomElementRemoteAppEntry(
+				customElementHTMLName = "vanilla-counter",
+				customElementName = "Vanilla Counter",
+				customElementURL = "http://remote-component-test.wincent.com/packages/vanilla-counter/index.js");
+		}
+
+		task ("Delete Vanilla Counter and check that has been deleted") {
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.deleteRemoteApp(tableEntry = "${customElementName}");
+
+			RemoteApps.viewTableEntryNameNotPresent(entryName = "${customElementName}");
```

Comment:
Separate this assertion line to it's own task.

As a rule of thumb, if you find yourself inserting "and" in the task, then we're probably adding too many actions within the task. There are exceptions to this but I think this is a good one to separate.

As a sidenote, I'd like to follow up with a refactor to change this macro to assertTableEntry..., it'll be more clear where the assertion is on the testcase level.